debian/copyright

```diff
@@ -0,0 +1,20 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: wasm-toolchain
+Source: <https://github.com/WebAssembly>
+
+Files: *
+Copyright: Copyright 2016 WebAssembly Community Group participants
+License: Apache
+
+License: Apache
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ .
+ http://www.apache.org/licenses/LICENSE-2.0
+ .
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
```

Comment:
LLVM is UIUC/NCSA license, and Emscripten is dual-licensed with MIT and LLVM licenses. Musl is... MIT? I think that's everything.